packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -2893,15 +2893,21 @@ Iterable<DiagnosticsNode> transformDebugCreator(Iterable<DiagnosticsNode> proper
 /// Transform the input [DiagnosticsNode].
 ///
 /// Return null if input [DiagnosticsNode] is not applicable.
-Iterable<DiagnosticsNode>? _parseDiagnosticsNode(DiagnosticsNode node) {
+Iterable<DiagnosticsNode>? _parseDiagnosticsNode(
+  DiagnosticsNode node,
+  Iterable<DiagnosticsNode> properties,
```

Comment:
rather than passing in all the properties which is confusing how about we just pass in the ErrorSummary for the error.
it would be cheap enough to extract that in the previous method that I'm not worried about extracting it for all errors.

Reply:
done